scripts/validate-verification-api.ts

```diff
@@ -0,0 +1,93 @@
+import { go } from '@api3/promise-utils';
+import { z } from 'zod';
+
+import { type Chain, CHAINS } from '../src/index';
+
+const resultSchema = z.object({
+  chainname: z.string(),
+  chainid: z.string(),
+  blockexplorer: z.string(),
+  apiurl: z.string(),
+  status: z.number(),
+  comment: z.string(),
+});
+
+const etherscanV2SupportListSchema = z.object({
+  comments: z.string(),
+  totalcount: z.number(),
+  result: z.array(resultSchema),
+});
+
+const ETHERSCAN_API_URL = 'https://api.etherscan.io/v2/chainlist';
+
+async function main() {
+  const warnings: string[] = [];
+  const errors: string[] = [];
+
+  const goResponse = await go(async () => fetch(ETHERSCAN_API_URL));
+
+  if (!goResponse.success || !goResponse.data) {
+    throw new Error('Etherscan API is not reachable');
+  }
+
+  const response = goResponse.data;
+
+  if (!response.ok) {
+    throw new Error(`Etherscan API responded with status: ${response.status}`);
+  }
+
+  const data = (await response.json()) as z.infer<typeof etherscanV2SupportListSchema>;
```

Comment:
```suggestion
  const parseData = etherscanV2SupportListSchema.safeParse(await response.json());
  if (!parseData.success) {
    const errors = parseData.error.issues.map((issue) => `  path: '${issue.path.join('.')}' => '${issue.message}' `);
    throw new Error(`Etherscan API response schema mismatch:\n${errors.join('\n')}\n`);
  }
  const { data } = parseData;
```

Actually, I meant more like this.